.github/workflows/kickstart-tests.yml

```diff
@@ -55,10 +55,12 @@ jobs:
           REF_DEVEL='f'$FEDVERS'-devel'
           REF_RELEASE='f'$FEDVERS'-release'
 
-          if [ "$TARGET_BRANCH" == "$REF_DEVEL" ] || [ "$TARGET_BRANCH" == "$REF_RELEASE" ] ; then
-            echo "::set-output name=lorax_build_container::fedora:$FEDVERS" ;
-          elif [ "$TARGET_BRANCH" == "master" ] ; then
-            echo "::set-output name=lorax_build_container::fedora:rawhide" ;
+          if [ "$TARGET_BRANCH" == "$REF_DEVEL" ] || [ "$TARGET_BRANCH" == "$REF_RELEASE" ]; then
+            echo "::set-output name=lorax_build_container::fedora:$FEDVERS";
+          elif [ "$TARGET_BRANCH" == "master" ]; then
+            echo "::set-output name=lorax_build_container::fedora:rawhide";
+          elif [ "$TARGET_BRANCH" == "rhel-9" ]; then
+            echo "::set-output name=lorax_build_container::registry-proxy.engineering.redhat.com/rh-osbs/ubi9:latest";
```

Comment:
These changes are not needed at all, you can specify the container inline in the "duplicated" RHEL-9-specific step below.